gui/widgets/bandview.cpp

```diff
@@ -19,13 +19,17 @@
 #include <tbb/blocked_range2d.h>
 #include <tbb/parallel_for.h>
 
+#include "qmath.h"
+
 #include "gerbil_gui_debug.h"
+#include <QGraphicsPixmapItem>
+#include <QDebug>
 
 BandView::BandView()
 	: // note: start with invalid curLabel to trigger proper initialization!
-	  cacheValid(false), cursor(-1, -1), lastcursor(-1, -1), curLabel(-1),
-	  overlay(0), showLabels(true), singleLabel(false), holdLabel(false),
-	  ignoreUpdates(false),
+      cacheValid(false), cursor(-1, -1), lastcursor(-1, -1), curLabel(-1),
+      overlay(0), showLabels(true), holdLabel(false), selectedLabels(0),
```

Comment:
Is holdLabel still functioning? It shouldn't as we now click to select labels, not hover.